mffpy/tests/test_reader.py

```diff
@@ -202,3 +202,10 @@ def test_epochs_averaged(mffpath_4, idx, name, expected):
         val = getattr(read_epoch, key)
         assert val == exp, f"""
         epochs[{idx}][{key}] = {val} [should be {exp}]"""
+
+
+def test_mff_flavor(mffpath, mffpath_2, mffpath_3, mffpath_4):
+    assert Reader(mffpath).mff_flavor == 'continuous'
+    assert Reader(mffpath_2).mff_flavor == 'segmented'
+    assert Reader(mffpath_3).mff_flavor == 'continuous'
+    assert Reader(mffpath_4).mff_flavor == 'averaged'
```

Comment:
I'm not sure if we need to call it `mff_flavor`.  Wouldn't "flavor" suffice?  (I do like flavor though; you could call continous "vanilla" instead :))

Reply:
Are you implying that segmented and averaged files are more interesting than continuous?? lol

Reply:
Yeah, good suggestion. `mff_flavor` is a bit redundant since were talking about a `Reader` object.